src/web/src/components/SignUp/Schema/FormSchema.tsx

```diff
@@ -0,0 +1,57 @@
+import * as Yup from 'yup';
+
+import formModels from './FormModel';
+
+const {
+  firstName,
+  lastName,
+  displayName,
+  githubUsername,
+  github,
+  githubOwnership,
+  feeds,
+  allFeeds,
+  blogUrl,
+  blogOwnership,
+} = formModels;
+
+// Each signup step has one validation schema
+export default [
+  // First step has no validation logic
```

Comment:
This array is confusing.  Can I suggest you name each of these or at the very least put a comment above each section and name it?

```js
// Comment explaining what step 1 is
const step1 = Yup.object().shape({}),

// Comment explaining what step 2 is
const step2 = ...
...

export default[
  step1,
  step2,
  ...
];
```

or

```js
export default[
  // Explain step 1
  Yup.object().shape({}),

  // Explain step 2
  ...

 // Explain step 3
 ...
];
```

Reply:
Done